data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,86 @@ mission "Station Exploration Abandon Ship"
 			label fine
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
+mission "Hai Pilgrimage"
+	name "Hai spiritual leader to <planet>"
+	description "Bring a Hai spiritual leader to <destination>. He will pay you <payment>."
+	minor
+	clearance "You inform the authorities of the passenger you are transporting, and they quickly allow you to land."
+	source
+		government "Hai"
+		not attributes "station"
+	destination
+		attributes "unfettered"
+	passengers 1
+	to offer
+		random < 10
+		or
+			has "outfit: Quantum Keystone"
+			has "outfit: Quantum Key Stone"
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`It is an unusually foggy day on <origin>; the air is so thick with moisture that you cannot see more than a few paces in front of you. Perhaps that's why you're so startled when you turn around and see a Hai standing right next to you. He is so close you can see dewdrops on his whiskers, and you immediately notice the prominent jewelry that he is wearing. He appears to be quite old, judging by his stooped-over posture and the white fur around his muzzle.`
+			branch remnant
+				has "outfit: Quantum Key Stone"
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+				goto choice
+			label remnant
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "I have noticed that the Keystone your ship bears is from beyond the wormhole. If your people have access to Keystones as well, then I hope they too become wise to its power. Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
```

Comment:
I love the idea but is the remnant keystons in evidence enough for him to see it, or would he have to sense it?
@Zitchas what can you tell us?

Reply:
The Hai should only react to their own Keystones, which come with certificates of authenticity and applicable display cases.
They should be ignorant of/suspicious of keystones from other sources.

Reply:
Without getting into too much nitty gritty detail of how it where it's is mounted, the main difference is likely to be the fact that the Remnant use a different mount, and to a Hai it would look much more industrial and utilitarian than their own. Am interesting option might be if he accepts it, but expresses suspicion and concern that someone is making keystones without due reverence. Tinned with appreciation already included here. Sort of a "it is nice but..." Sorry if way. Maybe with encouragement to get a "proper" keystone, but accepting it for now.

What about if someone has one of each?

Reply:
There are two other keystones:
Shield Beetle Pendant
Tree Skeleton Key Stone

Reply:
Maybe the hai pretends to sense things but just uses a device to detect if there is a keystone or not?

Reply:
My thinking is that the spaceport authorities keep a register of who does/does not have one, so that they can direct inquiries from applicably wealthy/superstitious individuals.
This being because I'd like to have luxury transport jobs on the job board (I don't think this exists yet?) that are exclusively available to captains with keystones.

Reply:
I added a bunch of branching options for the other kinds of keystones, and took into account Grey and Zitchas' feedback regarding Hareek's response to a Remnant keystone.
Frankly I have no idea what to do if someone has multiple kinds of keystones.

Reply:
I changed it so that it defaults to the Hai keystone dialog if you have that installed along with some other kind of keystone.

Reply:
>  This being because I'd like to have luxury transport jobs on the job board (I don't think this exists yet?) that are exclusively available to captains with keystones.
The "transport wealthy passengers" mission requires both luxury accomodations and hai keystones.